Allow mutiple feedback messages (#21)
uon-nuget/UoN.AspNetCore.FeedbackMessage: README.md

- Alerts can include a "close" button, to dismiss them. This works out the box with [Bootstrap 4], or can otherwise be implemented separately.
- Can be used for static alerts as well as Session based `TempData` driven alerts.
- Can be used via AJAX to build alert markup without reloading the page.

> - Can show multiple alearts at once.

Reviewer: typo: alerts*
```suggestion
- Can show multiple alerts at once.
```

It provides the following code items:

- A model class representing a configurable Feedback Message.

> - Extension methods for setting and getting `FeedbackMessageModel`s at `TempData["FeedbackMessage"]`.

Reviewer: ```suggestion
- Extension methods for setting, getting and clearing multiple `FeedbackMessageModel`s at `TempData["FeedbackMessage"]`.
```

> - A View for rendering a `List<FeedbackMessageModel>` as a [Bootstrap 4] style alert.

Reviewer: would probably write this in regular english, and correct the fact it renders multiple alerts, not the whole list in a single alert
```suggestion
- A View for rendering a list of `FeedbackMessageModel`s with a [Bootstrap 4] style alert per model
```

1. Acquire the library via one of the methods above.
1. Ensure the [ASP.NET Core Session Middleware] is configured in your project.

> 1. Use `this.AddFeedbackMessage()` inside an MVC Controller method for every feedback message you want to display.

Reviewer: also mention `this.ClearFeedbackMessages()` around here?

```suggestion
1. Use `this.AddFeedbackMessage()` inside an MVC Controller method for every feedback message you want to display.
  - optionally use `this.ClearFeedbackMessages()` inside an MVC Controller method to remove all currently set messages.
```

1. Acquire the library via one of the methods above.
1. Ensure the [ASP.NET Core Session Middleware] is configured in your project.
1. Use `this.AddFeedbackMessage()` inside an MVC Controller method for every feedback message you want to display.

>   - optionally use `this.ClearFeedbackMessages()` inside an MVC Controller method to remove all currently set messages.

Reviewer: my suggestion's indenting was wrong

```suggestion
    - optionally use `this.ClearFeedbackMessages()` inside an MVC Controller method to remove all currently set messages.
```
